raiden-cli/src/routes/api.v1.ts

```diff
@@ -19,15 +19,24 @@ export function makeApiV1Router(this: Cli): Router {
   router.use('/testing', makeTestingRouter.call(this));
 
   router.get('/version', (_request: Request, response: Response) => {
+    response.json({ version: Raiden.version });
+  });
+
+  router.get('/contracts', (_request: Request, response: Response) => {
+    const contracts = this.raiden.contractsInfo;
     response.json({
-      version: Raiden.version,
+      contracts_version: Raiden.contractVersion,
+      token_network_registry_address: contracts.TokenNetworkRegistry.address,
+      secret_registry_address: contracts.SecretRegistry.address,
+      service_registry_address: contracts.ServiceRegistry.address,
+      user_deposit_address: contracts.UserDeposit.address,
+      monitoring_service_address: contracts.MonitoringService.address,
+      one_to_n_address: '',
     });
   });
 
   router.get('/address', (_request: Request, response: Response) => {
-    response.json({
-      our_address: this.raiden.address,
-    });
+    response.json({ our_address: this.raiden.address });
```

Comment:
Has formatting changed? I hate it when formatting is not deterministic. :expressionless:

Reply:
It didn't change, but prettier's recommended rules allow small objects to be either declared in one line or split lines. I just wanted to reduce the line counter for something so simple. If it becomes bigger, with more properties making it cross 99char line limit, then it must be broken.